doc/src/modules/physics/mechanics/masses.rst

```diff
@@ -40,15 +40,89 @@ analysis of points separate from their association with masses.
 Inertia
 =======
 
-See the Inertia (Dyadics) section in 'Advanced Topics' part of
-:mod:`sympy.physics.vector` docs.
+Inertia consists out of two parts: a quantity and a reference. The quantity is
+expressed as a :class:`Dyadic<sympy.physics.vector.dyadic.Dyadic>` and the
+reference is a :class:`Point<sympy.physics.vector.point.Point>`. The
+:class:`Dyadic<sympy.physics.vector.dyadic.Dyadic>` can be defined as the outer
+product between two vectors, which returns the juxtaposition of these vectors.
+For further information, please refer to the :ref:`Dyadic` section in the
+advanced documentation of the :mod:`sympy.physics.vector` module. Another more
+intuitive method to define the
+:class:`Dyadic<sympy.physics.vector.dyadic.Dyadic>` is to use the
+:func:`~.inertia` function as described below in the section
+'Inertia (Dyadics)'. The :class:`Point<sympy.physics.vector.point.Point>` about
+which the :class:`Dyadic<sympy.physics.vector.dyadic.Dyadic>` is specified can
+be any point, as long as it is defined with respect to the center of mass. The
+most common reference point is of course the center of mass itself.
+
+The inertia of a body can be specified using either an :class:`~.Inertia` object
+or a ``tuple``. If a ``tuple`` is used, then it should have a length of two,
+with the first entry being a :class:`Dyadic<sympy.physics.vector.dyadic.Dyadic>`
+and the second entry being a :class:`Point<sympy.physics.vector.point.Point>`
+about which the inertia dyadic is defined. Internally this ``tuple`` gets
+converted to an :class:`~.Inertia` object. An example of using a ``tuple`` about
+the center of mass is given below in the 'Rigid Body' section. The
+:class:`~.Inertia` object can be created as follows.::
+
+   >>> from sympy.physics.mechanics import ReferenceFrame, Point, outer, Inertia
+   >>> A = ReferenceFrame('A')
+   >>> P = Point('P')
+   >>> Inertia(P, outer(A.x, A.x))
+   ((A.x|A.x), P)
```

Comment:
I'm guessing this will need pretty printers and latex printers. Or does NamedTuple print nicely with SymPy's functionality.

Reply:
I expect that it doesn't print in the format how you would like it.

Reply:
Maybe at least open and issue for printing of this object as a todo.

Reply:
I'll do it, when this PR is merged.